Backend/router/v1/report.ctrl.js

```diff
@@ -0,0 +1,67 @@
+const models = require('../../models');
+
+const create = (req, res) => {
+  models.report.create({
+    what: req.body.data[0].what,
+    location: req.body.data[0].location,
+    time: req.body.data[0].time,
+    who: req.body.data[0].who,
+    details: req.body.data[0].details,
+  }).then(result => {
+    console.log(result);
+    console.log("success!");
+  });
+  res.status(201);
+  res.send("good");
+}
+
+const list = (req, res) => {
+  models.report.findAll().then((result) => {
+    res.json(result);
+  }).catch((err) => {
+    console.log("error!!!!!!!!!!");
```

Comment:
If you want to show to the user, can you make it more meaningful instead of logging error?
For example, log err.